x/muxer/muxer.go

```diff
@@ -127,16 +129,16 @@ func (m *muxer) CheckConfig(ctx context.Context, req *rpc.CheckRequest) (*rpc.Ch
 		// Add missing inputs, but don't override existing inputs.
 		for k, v := range r.A.GetInputs().GetFields() {
 			existingValue, has := inputs.Fields[k]
-			if has {
-				// If different servers return different check values,
-				// signal an error. We don't know how to deal with that.
-				if proto.Equal(existingValue, v) {
-					continue
-				}
-				return nil, fmt.Errorf("config value '%s' mismatch between servers %d and a previous server",
-					k, i)
+			if has && !proto.Equal(existingValue, v) {
+				// If different servers return different values, pick arbitrarily.
```

Comment:
is there a way for the user to pick explicitly so they don't hit an error later on? disregard if that is an unlikely scenario.

Reply:
We have only one known use case for this code in production, muxed V3/PF providers. They really should agree.. I'm not sure the users would know what to do with this... Current code picks up the conservative choice (V3) that has been around in production more, that's the right call I think.